rcl/test/rcl/test_get_actual_qos.cpp

```diff
@@ -190,9 +231,19 @@ TEST_P_RMW(TestSubscriptionGetActualQoS, test_subscription_get_qos_settings) {
   EXPECT_EQ(
     qos->durability,
     parameters.qos_expected.durability);
+  EXPECT_GE(
+    qos->deadline,
+    parameters.qos_expected.deadline);
+  // lifespan is not a concept in subscriptions
+  // EXPECT_GE(
+  //   qos->lifespan,
+  //   parameters.qos_expected.lifespan);
```

Comment:
Delete commented code. Comment in L237 should stay.

Reply:
Done.